proposals/0017-conforming-literals.md

```diff
@@ -82,6 +82,100 @@ adopting Clang).
 For that reason, this feature proposal targets HLSL 202x, with support for the
 new literal behavior in DXC.
 
+### Experimentation
+
+In writing this proposal a key thesis was proposed that the impact of this
+change on developers would be minimal because DXC's behavior for DXIL is
+undocumented, complex, and different from other shader compilers (including
+DXC's SPIR-V generator). The last point is important and relevant because a
+large percentage of shader code is compiled with multiple different shader
+compilers which implement different language semantics. Because of this, it is
+believed that shaders need to be resilient to subtle behavioral differences like
+the ones this change would cause.

```

Comment:
```suggestion
believed that many existing shaders are engineered to be resilient to subtle behavioral differences like
the ones this change would cause.
```